src/databricks/labs/remorph/cli.py

```diff
@@ -74,21 +77,28 @@ def transpile(
 
 
 @remorph.command
-def reconcile(w: WorkspaceClient, recon_conf: str, conn_profile: str, source: str, report: str):
+def reconcile(w: WorkspaceClient, source: str, report: str):
     """reconciles source to databricks datasets"""
     logger.info(f"user: {w.current_user.me()}")
-    if not os.path.exists(recon_conf) or recon_conf in {None, ""}:
-        raise_validation_exception(f"Error: Invalid value for '--recon_conf': Path '{recon_conf}' does not exist.")
-    if not os.path.exists(conn_profile) or conn_profile in {None, ""}:
-        raise_validation_exception(f"Error: Invalid value for '--conn_profile': Path '{conn_profile}' does not exist.")
-    if source.lower() not in "snowflake":
-        raise_validation_exception(f"Error: Invalid value for '--source': '{source}' is not one of 'snowflake'. ")
-    if report.lower() not in {"data", "schema", "all"}:
+    installation = Installation.current(w, 'remorph')
+    table_recon = installation.load(TableRecon)
+
+    if source.lower() not in ["databricks", "snowflake", "oracle"]:
+        raise_validation_exception(
+            f"Error: Invalid value for '--source': '{source}' is not one of 'databricks', 'snowflake', 'oracle'. "
+        )
+    if report.lower() not in {"data", "schema", "all", "hash"}:
         raise_validation_exception(
-            f"Error: Invalid value for '--report': '{report}' is not one of 'data', 'schema', 'all' "
+            f"Error: Invalid value for '--report': '{report}' is not one of 'data', 'schema', 'all' , 'hash'"
         )
 
-    recon(recon_conf, conn_profile, source, report)
+    spark = _get_spark_session(w)
+
+    recon(w, spark, table_recon, SQLGLOT_DIALECTS.get(source.lower()), report)
```

Comment:
```suggestion
    recon_id = recon(w, spark, table_recon, SQLGLOT_DIALECTS.get(source.lower()), report)
    print(json.dumps(recon_id))
```

Reply:
You will need to update `labs.yml` as well.

Reply:
done.